dist/add-markers.js

```diff
@@ -221,7 +221,16 @@ Fliplet.InteractiveMap.component('add-markers', {
     }
   },
   methods: {
-    resetSelectFields: function resetSelectFields() {
+    resetSelectFields: function resetSelectFields(columns) {
+      if (columns && columns.length) {
+        this.markerNameColumn = columns[0];
+        this.markerMapColumn = columns[1];
+        this.markerTypeColumn = columns[2];
+        this.markerXPositionColumn = columns[3];
+        this.markerYPositionColumn = columns[4];
```

Comment:
`columns` having `length` doesn't mean there would be 5 entries in the array. I think it's best that each of the variables get a default value.

`this.markerTypeColumn = columns[2] || '';` for example. The default value would depend on the variable. It coudl be a string, number, empty array, false, null etc.